server/package.json

```diff
@@ -1,17 +1,31 @@
 {
   "scripts": {
-    "dev": "ts-node-dev --respawn --transpile-only src/index.ts"
+    "dev": "nodemon -x ts-node src/index.ts",
+    "lint": "eslint --ext ts ./src",
+    "lint:fix": "yarn lint --fix",
+    "prettier": "prettier --ignore-path .gitignore --write src"
   },
   "dependencies": {
-    "@types/express": "^4.17.21",
-    "@types/node": "^20.11.24",
     "cors": "^2.8.5",
     "express": "^4.18.3",
     "pg": "^8.11.3",
-    "typescript": "^5.3.3"
+    "reflect-metadata": "^0.2.2",
+    "sequelize": "6.37.2",
+    "sequelize-typescript": "2.1.6",
```

Comment:
We're using sequelize-typescript instead of direct sequel since [direct Typescript support is not fully finished](https://sequelize.org/docs/v6/other-topics/typescript/) and is available in sequelize-typescript by the sequelize team. [v7 of sequelize is in alpha but is unstable](https://sequelize.org/docs/v7/models/model-typing/), until it is officially released we can use sequelize-typescript.